app/assets/javascripts/add_date_field.js.coffee

```diff
@@ -10,7 +10,18 @@
         validateOnBlur: false
       })
 
+    date_multiple_default = () ->
```

Comment:
Related to comment below, take `selector` as an argument to this method and call .datetimepicker on it.

Something like;

```
date_multiple_default = (selector) ->
  selector.datetimepicker({
...
```